src/main/java/cz/student/jankocabek/TaskManager.java

```diff
@@ -1,9 +1,198 @@
+                default -> {
+                    System.out.printf("%sInvalid option!%s\n", RED_BOLD_BRIGHT, RESET);
+                    System.out.printf("%sEnter correct options or \"%s?%s\" for menu%s\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, BLUE_BOLD_BRIGHT, RESET);
+                }
+            }
+        } while (true);
+    }
+
+    //loading tasks from file checking errors in case of missing file try to create new for smooth continue.
+    private static void loadTasks() {
+        final Path pathToFile = Paths.get("tasks.csv");
+        tasks = new String[0][];//here will be tasks data
+        System.out.printf("%sSystem will now read all the tasks from the task.csv file%s\n", YELLOW_BOLD_BRIGHT, RESET);
+        try {
+            if (Files.exists(pathToFile)) {
+                tasks = listTo2DArr(Files.readAllLines(pathToFile));
+                System.out.printf("%sTask were successfully loaded from tasks.csv%s\n", GREEN_BOLD_BRIGHT, RESET);
+            } else {
+                System.out.printf("%sFile tasks.csv not found!%s\n", RED_BOLD_BRIGHT, RESET);
+                System.out.printf("%sProgram will try create new file%s\n", YELLOW_BOLD_BRIGHT, RESET);
+                Files.createFile(pathToFile);
+                System.out.printf("%sFile tasks.csv successfully created%s\n", GREEN_BOLD_BRIGHT, RESET);
+            }
+        } catch (IOException e) {
+            System.err.println("Error while reading tasks.csv file");
+            e.printStackTrace(System.err);
+            System.err.println("Program will try create new file");
+            try {
+                Files.createFile(pathToFile);
+            } catch (IOException ex) {
+                throw new RuntimeException(ex);
+            }
+            System.out.printf("%sFile tasks.csv successfully created%s\n", GREEN_BOLD_BRIGHT, RESET);
+        }
+    }
 
+    private static void displayMenu() {
+        System.out.printf("\n%s#  Welcome to the Task Manager  #\n", GREEN_BOLD_BRIGHT);
+        System.out.print("#                               #\n");
+        System.out.printf("#   %s1. Add a new task:  %sadd     %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#   %s2. Remove a task:   %sremove  %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#   %s3. Show all tasks:  %slist    %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#   %s4. Save and Exit:   %sexit    %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#################################%s\n", RESET);
     }
 
-}
+    private static void showTasks() {
+        System.out.printf("\n%sTask List:%s\n", GREEN_BOLD_BRIGHT, RESET);
+        String[] task;
+        for (int i = 0; i < tasks.length; i++) {//this loop controlling if task is important or not for different colors in output
+            task = tasks[i];
```

Comment:
CLEAN-CODE: promennou task bych inicializoval pouze v miste pouziti a to je uvnitr tohoto cyklu

```suggestion
final var task = tasks[i];
```